dicer/mappers/patient_mapper.py

```diff
@@ -0,0 +1,90 @@
+from typing import Dict, List
+
+from transmart_loader.transmart import Patient as TLPatient, IdentifierMapping as TLIdentifierMapping, \
+    Relation as TLRelation, RelationType as TLRelationType, Visit as TLVisit
+
+from dicer.mappers.mapper_helper import DataInconsistencyException
+from dicer.transmart import PatientDimensionElement, Relation, Value, VisitDimensionElement
+
+
+class PatientMapper:
+    """
+    Map patient dimension elements and patient relations from query results to transmart-loader objects
+    """
+
+    def __init__(self):
+        self.patient_id_to_patient_identifier: Dict[int, str] = {}
```

Comment:
This could also be `self.patient_id_to_patient: Dict[int, TLPatient]`. The resulting list of patients can then be obtained with `self.patient_id_to_patient.values()`

Reply:
right! That simplifies many things, I'm not sure why I did it in this over-complicated way